san/graphql.py

```diff
@@ -5,45 +5,47 @@
 
 
 def execute_gql(gql_query_str):
-  headers = {}
-  if ApiConfig.api_key:
-    headers = {'authorization': "Apikey {}".format(ApiConfig.api_key)}
-
-  response = requests.post(
-    SANBASE_GQL_HOST,
-    json={
-        'query': gql_query_str},
-    headers=headers)
-
-  if response.status_code == 200:
-    return __handle_success_response__(response, gql_query_str)
-  else:
-    raise SanError(
-      "Error running query. Status code: {}.\n {}".format(
-        response.status_code,
-        gql_query_str))
+    headers = {}
+    if ApiConfig.api_key:
+        headers = {'authorization': "Apikey {}".format(ApiConfig.api_key)}
+
+    response = requests.post(
+        SANBASE_GQL_HOST,
+        json={
+            'query': gql_query_str},
+        headers=headers)
+
+    if response.status_code == 200:
+        return __handle_success_response__(response, gql_query_str)
+    else:
+        raise SanError(
+            "Error running query. Status code: {}.\n {}".format(
+                response.status_code,
+                gql_query_str))
+
 
 def __handle_success_response__(response, gql_query_str):
-  if __result_has_gql_errors__(response):
-    raise SanError(
-      "GraphQL error occured running query {} \n errors: {}".format(
-        gql_query_str,
-        response.json()['errors']))
-  elif __exist_not_empty_result(response):
-    return response.json()['data']
-  else:
-    raise SanError(
-      "Error running query. Status code: {}.\n {} \n errors: {}" .format(
-        response.status_code,
-        gql_query_str,
-        response.json()['errors']))
```

Comment:
Removing only this line here should fix the error handling (there is no `errors` here) - no further restructuring is needed.